src/module/display.js

```diff
@@ -0,0 +1,48 @@
+const renderToDo = () => {
+  const listItems = document.getElementById('list-items');
+
+  const newData = [
+    {
+      description: 'Wash Car',
+      completed: false,
+      index: 0,
+    },
+
+    {
+      description: 'Clean Clothes',
+      completed: false,
+      index: 1,
+    },
+
+    {
+      description: 'Running',
+      completed: false,
+      index: 2,
+    },
+    {
+      description: 'Travel',
+      completed: false,
+      index: 2,
+    },
+  ];
+
+  listItems.innerHTML = '';
+  newData.forEach((item) => {
```

Comment:
- [ ] [OPTIONAL] I highly suggest you display the list by order of their index to meet the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md). You can use the array `sort` method to achieve this. 😎️

**Project Requirements**
![image](https://user-images.githubusercontent.com/17243325/220849886-8c697a76-01b3-4136-962a-e74ca1690dd3.png)

**Example**
```
newData.sort((a, b) => a.index - b.index).forEach((item) => {
  ...
})
```